test/extensions/extensions_build_system.bzl

```diff
@@ -1,14 +1,14 @@
 load("//bazel:envoy_build_system.bzl", "envoy_benchmark_test", "envoy_cc_benchmark_binary", "envoy_cc_mock", "envoy_cc_test", "envoy_cc_test_binary", "envoy_cc_test_library")
-load("@envoy_build_config//:extensions_build_config.bzl", "EXTENSIONS")
+load("@envoy_build_config//:extensions_build_config.bzl", "DISABLED_BY_DEFAULT_EXTENSIONS", "EXTENSIONS")
 
 # All extension tests should use this version of envoy_cc_test(). It allows compiling out
 # tests for extensions that the user does not wish to include in their build.
-# @param extension_name should match an extension listed in EXTENSIONS.
+# @param extension_name should match an extension listed in EXTENSIONS or DISABLED_BY_DEFAULT_EXTENSIONS.
 def envoy_extension_cc_test(
         name,
         extension_name,
         **kwargs):
-    if not extension_name in EXTENSIONS:
+    if not extension_name in EXTENSIONS and not extension_name in DISABLED_BY_DEFAULT_EXTENSIONS:
```

Comment:
Nit: factor this out into some macro, e.g.
```
def extension_allowed_for_test(name):
  return extension_name in EXTENSIONS or extension_name in DISABLED_BY_DEFAULT_EXTENSIONS
```

Reply:
Done